dll/shellext/acppage/lang/hi-IN.rc

```diff
@@ -0,0 +1,49 @@
+/*
+ * PROJECT:     ACPPAGE Hindi Translation
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
```

Comment:
`GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)` seems to be the preferred style.